app/controllers/api/comments_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Api::Students::CommentsController < Api::BaseController
+class Api::CommentsController < Api::BaseController
 
   def create
```

Comment:
Since the controller hierarchy has changed, you'll need to update the `comment_params` helper method and probably the `create` method as well. Since a `Comment` needs a `commentable_id` (see below), that'll be one of the whitelisted attributes and another will be `commentable_type`.

The client-side will need to be updated as well (can you guess what needs to be changed?)!